src/data/actions/assets.js

```diff
@@ -6,8 +6,8 @@ export const requestAssetsSuccess = response => ({
   data: response.assets,
 });
 
-export const assetXHRFailure = response => ({
-  type: assetActions.ASSET_XHR_FAILURE,
+export const assetDeleteFailure = response => ({
+  type: assetActions.DELETE_ASSET_FAILURE,
```

Comment:
We may not even want to do this, by the way. It's just an idea that would prevent us from having to write the same kind of boilerplate code over and over.

Reply:
I wanted to separate this out because regardless, we'd need to differentiate the types of errors. I could potentially see the differentiation localized somewhere in redux (e.g. setting the message, error severity, etc.), but ultimately the frontend does need to know about the specifics, especially when we account for i18n. The text will need to be marked for translation in the view.

Anyway, I think we might be able to achieve all our goals with some tweaking and probably easier to get on the same page in person. Look for our a meeting, Dahlia!